Makefile

```diff
@@ -60,7 +63,8 @@ serve-assets:
 
 .PHONY: serve-all
 serve-all:
-	./node_modules/.bin/concurrently --kill-others -r "./scripts/serve.sh" "yarn --cwd ./themes/default/theme run start"
```

Comment:
What's the rationale behind removing
`./node_modules/.bin/concurrently --kill-others` ?

Reply:
Just trying to drop the dependency. I can put it back: https://github.com/pulumi/registry/pull/6201/commits/29ca99d7df6a8c8348360ad57e682f1965c82d1e.